docs/resources/aws_elasticache_replication_group.md

```diff
@@ -0,0 +1,78 @@
+---
+title: About the aws_elasticache_replication_group Resource
+---
+
+# aws_elasticache_replication_group
+
+Use the `aws_elasticache_replication_group` InSpec audit resource to test the properties of a single Amazon ElastiCache replication group.
+
+## Syntax
+
+An `aws_elasticache_replication_group` resource block declares the tests for a single Amazon ElastiCache replication group by `replication_group_id`.
+
+    describe aws_elasticache_replication_group(replication_group_id: 'my-replication-group-123') do
+      it { should exist }
+    end
+
+The value of the `replication_group_id` can be provided as a string.  
+
+    describe aws_elasticache_replication_group('my-replication-group-123') do
+      it { should exist }
+    end
+
+#### Parameters
+
+The ElastiCache replication group ID is required.
+
+##### replication\_group\_id _(required)_
```

Comment:
I missed a couple headings. Otherwise, LGTM.

```suggestion
### Parameters

The ElastiCache replication group ID is required.

#### replication\_group\_id _(required)_
```